.github/workflows/docker.yaml

```diff
@@ -88,5 +88,6 @@ jobs:
         cache-from: type=registry,ref=${{ env.IMAGE_CACHE }}
         cache-to: type=registry,ref=${{ env.IMAGE_CACHE }},mode=max
         build-args: |
+          BRANCH_NAME=${{ github.head_ref || github.ref_name }}
```

Comment:
can we do this based on sha instead of branch, just to match how composer does it?

Reply:
no, at least not in a way I could think of. Maybe there's another way? If we do it based on commit hash then the git clone command (L16 in Dockerfile) would be different and the docker cache would always be invalidated even if setup.py has no changes.

Reply:
ah i suppose thats true...ok